lib/pages/home_page.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
```

Comment:
All your modifications should be located inside the lib/sht/ folder. Please move this file